Demo/Sources/Views/LongPressPlaybackView.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import PillarboxPlayer
+import SwiftUI
+
+#if os(iOS)
+private struct LongPressView<Content>: View where Content: View {
+    @GestureState private var isLongPressing = false
+    @State private var timer: Timer?
+
+    let minimumPressDuration: TimeInterval
+    @ViewBuilder let content: () -> Content
```

Comment:
```suggestion
    let content: () -> Content
```

`@ViewBuilder` is superfluous when the initializer has been explicitly implemented if I am not mistaken.